Test/RivetTest/Import-RivetTest.ps1

```diff
@@ -68,12 +71,19 @@ $importModuleParams = @{ }
 
 if( $Force -and $loadedModule )
 {
-    # Remove so we don't get errors about conflicting type data.
-    Remove-Module -Name 'RivetTest' -Verbose:$false -WhatIf:$false
+    & {
+        $VerbosePreference = 'SilentlyContinue'
+        $WhatIfPreference = $false
+        # Remove so we don't get errors about conflicting type data.
+        Remove-Module -Name 'RivetTest' -Verbose:$false -WhatIf:$false

```

Comment:
Just curious here - Do downstream functions/scripts/etc., use the `$WhatIfPreference` instead of honoring the `WhatIf` switch of an upstream caller?

Reply:
Anything inside or called by anything inside the block should honor what's in that block.